### ISSUE TITLE
Unpack zip file

### ISSUE THREAD

**Issue by evrardts** [tadasauciunas/libreoffice-lambda-layer-brotli#1]
Hi, thanks a lot for the package. I am using the package 6.4.0.3.

Quick question, how do you unpack it? 

I tried: 

const {unpack} = require('@shelf/aws-lambda-brotli-unpacker');

const inputPath = '/opt/layer.tar.br';
const outputPath = '/tmp/instdir/program/soffice.bin';

await unpack({inputPath, outputPath});

But I get this error with this output path: 

"Error: ENOENT: no such file or directory, chmod '/tmp/instdir/program/soffice.bin'"

Any idea what the output path needs to be? 

Thanks!

**Comment by tadasauciunas** [comment]
Did you publish the layer with the .zip file included in this repository ? Can you verify that /opt/layer.tar.br exists before unpacking ?

**Comment by evrardts** [comment]
Thanks for coming back so quickly. Yes I published the layer and verified that layer.tar.br is on opt repo: 

<img width="689" alt="image" src="https://user-images.githubusercontent.com/30987204/103886261-8722ca80-50e1-11eb-8949-9f989858e5ba.png">

Any idea what else it could be?

Thanks again for your help!

**Comment by tadasauciunas** [comment]
Try renaming `layer.tar.br.zip` to `lo.tar.br.zip` when publishing the layer

**Comment by tadasauciunas** [comment]
Or make sure that the the file from the layer ends up being called `lo.tar.br` inside the system in other ways, cause that's what the shelfio package expects. See following link for reference
https://github.com/shelfio/aws-lambda-libreoffice/blob/6a48d1b5bfd30fb1e4b3b7b277599c62b91b6575/src/convert.ts#L19

**Comment by evrardts** [comment]
Thanks for the advice. I renamed to lo.tar.br but it still doesn't work. I tried using the @shelf/aws-lambda-brotli-unpacker package to unpack the layer and also the convertTo function directly but none is working. 
Maybe my whole approach is not correct, here is what I did:

1. I downloaded your package : https://github.com/tadasauciunas/libreoffice-lambda-layer-brotli/blob/master/6.4.0.3/lo.tar.br.zip
2. Put the zip file within a bucket
3. Created a layer in lambda function and connected the bucket to it
4. Attached the layer to the function
5. And then within the function unpack the layer. 

What is strange is that if I use the arn (arn:aws:lambda:eu-central-1:764866452798:layer:libreoffice-brotli:1) provided here (https://github.com/shelfio/libreoffice-lambda-layer) my code is working correctly. I just can't convert a docx with a table of content into a pdf. That is why I wanted to try out your package but I am really unable to unpack it correctly. 

Thanks again for the help.
